ibis/backends/flink/tests/test_ddl.py

```diff
@@ -17,6 +17,34 @@
 except ImportError:
     Py4JJavaError = None
 
+_awards_players_schema = sch.Schema(
+    {
+        "playerID": dt.string,
+        "awardID": dt.string,
+        "yearID": dt.int32,
+        "lgID": dt.string,
+        "tie": dt.string,
+        "notes": dt.string,
+    }
+)
+
+_functiona_alltypes_schema = sch.Schema(
+    {
+        "id": dt.int32,
+        "bool_col": dt.bool,
+        "smallint_col": dt.int16,
+        "int_col": dt.int32,
+        "bigint_col": dt.int64,
+        "float_col": dt.float32,
+        "double_col": dt.float64,
+        "date_string_col": dt.string,
+        "string_col": dt.string,
+        "timestamp_col": dt.timestamp(scale=3),
+        "year": dt.int32,
+        "month": dt.int32,
+    }
```

Comment:
What's the point in moving these out?

Reply:
so the schema can be used in non-fixture situation (e.g., [here](https://github.com/ibis-project/ibis/pull/7640/files/caab87f1ab772095f05b6fadd541249ba0cb2052#diff-e7a6d6edeb616d3cc2b66f70924b3edf4b0bc51df2e2d653b9551030bac00cd2R181)). Is there a better way to do this though?

Reply:
there's not really a good solution for parametrizing fixtures -- there's a pytest plugin called `lazy_fixtures` which will let you pass the fixture as a variable to parametrize, but what you have here is a reasonable solution.